cmd/chihaya/config.go

```diff
@@ -52,3 +60,25 @@ func ParseConfigFile(path string) (*ConfigFile, error) {
 
 	return &cfgFile, nil
 }
+
+// CreateHooks creates instances of Hooks for all of the PreHooks and PostHooks
+// configured in a ConfigFile.
+func (cfg ConfigFile) CreateHooks() (preHooks, postHooks []middleware.Hook, err error) {
+	for _, hookCfg := range cfg.MainConfigBlock.PreHooks {
+		switch hookCfg.Name {
+		case "jwt":
+			jwtCfg, ok := hookCfg.Config.(jwt.Config)
```

Comment:
I don't think so, I just wrote this part super fast and didn't check it. I'll try it out later today.